packages/clay-form/src/Checkbox.tsx

```diff
@@ -81,6 +87,7 @@ const ClayCheckbox = React.forwardRef<HTMLInputElement, IProps>(
 				<label>
 					<input
 						{...otherProps}
+						aria-labelledby={id}
```

Comment:
I think the ideal here is to use the `aria-labelledby` property because using the id would be used incorrectly since both have different meanings here. This could also cause a breaking change because the id was passed along with the `otherProps` to the input.

My suggestion is to pass the `aria-labelledby={labelId}` in the TreeViewItem so that there would be no need for changes in the Checkbox.

Reply:
Thanks @matuzalemsteles! I'll proceed with this change.